script/integration/k8s/integration.yaml

```diff
@@ -296,6 +296,7 @@ data:
       region: "my-region"
       scheme: local
       signedUrls:
+        enabled: true
```

Comment:
If the default behavior is going to be false as per this https://github.com/flyteorg/flyteadmin/pull/283/files#diff-4149c9b64083f7b9e6680f7294d71f6ba58bc8da28a1189309eeea69d8ed07f3R63

Should we instead have integration test for false since that will be run without any config changes.
If possible we can have both the vairations in integration tests

Reply:
since changing configs in integration tests is a bit complicated I'll just remove this and adjust the tests accordingly to use the default behavior.